src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -54,7 +54,12 @@ foam.CLASS({
     {
       class: 'String',
       name: 'browseTitle',
-      expression: function(of) { return foam.String.pluralize(of.model_.label); }
+      expression: function(of) { return  of.model_.plural }
+    },
+    {
+      class: 'String',
+      name: 'browseSubtitle',
+      expression: function(of) { return "View all " + of.model_.plural.toLowerCase() +'.' }
```

Comment:
These would do better as factories. Expression logic can override code that is explicitly set on these props.

Reply:
Should be a space after the last +.
Expressions don't override explicitly set values. Once you set the value, the expression is deactivated.

Reply:
Pushing space change to https://github.com/Nauna/foam2/pull/82

Leaving as expression, which will update 'browseSubtitle' if 'of' changes, could be desirable if something is loading and not synchronized.